api/swagger.yaml

```diff
@@ -665,7 +665,17 @@ definitions:
               type: "string"
           IpcMode:
             type: "string"
-            description: "IPC namespace to use for the container."
+            description: |
+                    IPC sharing mode for the container. Possible values are:
+
+                    - `"none"`: own private IPC namespace, with /dev/shm not mounted
+                    - `"private"`: own private IPC namespace
+                    - `"shareable"`: own private IPC namespace, with a possibility to share it with other containers
+                    - `"container:<name|id>"`: join another (shareable) container's IPC namespace
+                    - `"host"`: use the host system's IPC namespace
```

Comment:
I haven't done that as I am not sure how swagger enum can work with `container:<ID>` notation (where `<ID>` is a variable part). I see a few options, but am not sure which one works:
 - omit the `container:<ID>` entrely;
 - specify is as `container:`, omitting the variable suffix;
 - specify is as `container`;
 - specify is as `container:<ID>`.

Glancing through swagger docs (https://swagger.io/docs/specification/enums/) didn't help.